server/test/services/cloud/ApplicantFileNameFormatterTest.java

```diff
@@ -11,4 +12,35 @@ public void formatFileUploadQuestionFilename() {
     assertThat(ApplicantFileNameFormatter.formatFileUploadQuestionFilename(1L, 2L, "3-4"))
         .isEqualTo("applicant-1/program-2/block-3-4/${filename}");
   }
+
+  @Test
+  public void isApplicantOwnedFileKey_isTrue() {
+    assertThat(
+            ApplicantFileNameFormatter.isApplicantOwnedFileKey(
+                "applicant-1/program-2/block-3-4/${filename}", 1L))
+        .isEqualTo(true);
+  }
+
+  @Test
+  public void isApplicantOwnedFileKey_isFalse() {
```

Comment:
Nitpick (optional): It took me a bit to figure out why this should return false. Maybe we can make the test name `isApplicantOwnedFileKey_applicantIdDoesNotMatch_isFalse` to make it clearer?